### PR TITLE
docs: finished draft section on semantic interoperability for chapter 2

### PR DESCRIPTION
**Pull request by WvandenBerg**
My goals for this text:
- Why it matters: easy to understand the barrier that semantic ambiguity forms to data sharing
- Why common vocabularies are important
- Why you need governance for these vocabularies
- A link to Linked Data technology

### CODE REVIEW

File: documentation/2_Context_of_the_International_Data_Spaces/2_5_Meaningful_data.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+The ability of IT systems to exchange data with unambiguous, shared meaning is called semantic interoperability. Semantic interoperability is an essential requirement for federated data networks such as IDS. It requires that data providers and data consumers in the network express their data offering or need using explicit reference to a common vocabulary. The commonality is important; if data is provided with references to a vocabulary that the receiving party is unfamiliar with, then the need to spend integration effort on their end remains. Vocabularies become common (shared) through the process of standardization.
```
Comment (clange) [inline documentation/2_Context_of_the_International_Data_Spaces/2_5_Meaningful_data.md:7]:
```suggestion
The ability of IT systems to exchange data with unambiguous, shared meaning is called _semantic interoperability_. Semantic interoperability is an essential requirement for federated data networks such as IDS. It requires that data providers and data consumers in the network express their data offering or need using explicit reference to a common vocabulary. The commonality is important; if data is provided with references to a vocabulary that the receiving party is unfamiliar with, then the need to spend integration effort on their end remains. Vocabularies become common (shared) through the process of standardization.
```
(just added emphasis)

File: documentation/2_Context_of_the_International_Data_Spaces/2_5_Meaningful_data.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+Many industries and other business ecosystems have turned to open standardization as a means to achieve semantic interoperability between its members. Open standardization means that members collaboratively maintain and develop semantic standards. It is a continuous balancing act between the need for strict uniformity to keep data consistent and easy to understand, and the need to accomodate for the fact that different organizations have different requirements for their data.
```
Comment (clange) [inline documentation/2_Context_of_the_International_Data_Spaces/2_5_Meaningful_data.md:9]:
```suggestion
Many industries and other business ecosystems have turned to open standardization as a means to achieve semantic interoperability between their members. Open standardization means that members collaboratively maintain and develop semantic standards. It is a continuous balancing act between the need for strict uniformity to keep data consistent and easy to understand, and the need to accommodate for the fact that different organizations have different requirements for their data. 
```
(minor spelling and grammar)

File: documentation/2_Context_of_the_International_Data_Spaces/2_5_Meaningful_data.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+Many industries and other business ecosystems have turned to open standardization as a means to achieve semantic interoperability between its members. Open standardization means that members collaboratively maintain and develop semantic standards. It is a continuous balancing act between the need for strict uniformity to keep data consistent and easy to understand, and the need to accomodate for the fact that different organizations have different requirements for their data. 
+
+This means there is often a limit to the level of semantic interoperability that can be achieved. Every member in a business ecosystem operates with a different world view. These differences arise from operating in different jurisdictions, in different domains, carrying out different business processes, serving different marktes, offering different services, and so on. High variety between members in business ecosystems means that any semantic standard for that community will have to allow for flexibility, which means some integration effort will remain necessary. Low variety allow for stricter semantic standards that bring more uniformity, allowing for more efficient data sharing and automatisation.
```
Comment (clange) [inline documentation/2_Context_of_the_International_Data_Spaces/2_5_Meaningful_data.md:11]:
```suggestion
This means there is often a limit to the level of semantic interoperability that can be achieved. Every member in a business ecosystem operates with a different world view. These differences arise from operating in different jurisdictions, in different domains, carrying out different business processes, serving different markets, offering different services, and so on. High variety between members in business ecosystems means that any semantic standard for that community will have to allow for flexibility, which means some integration effort will remain necessary. Low variety allows for stricter semantic standards that bring more uniformity, thus allowing for more efficient data sharing and automation.  
```
(minor linguistics)

File: documentation/2_Context_of_the_International_Data_Spaces/2_5_Meaningful_data.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+This means there is often a limit to the level of semantic interoperability that can be achieved. Every member in a business ecosystem operates with a different world view. These differences arise from operating in different jurisdictions, in different domains, carrying out different business processes, serving different marktes, offering different services, and so on. High variety between members in business ecosystems means that any semantic standard for that community will have to allow for flexibility, which means some integration effort will remain necessary. Low variety allow for stricter semantic standards that bring more uniformity, allowing for more efficient data sharing and automatisation.  
+
+In any case, governance has to be put in place to make sure that a semantic standard serves the needs of the community as best as possible and will remain doing so. How these governance processes can be organized is discussed in section 4.
```
Comment (clange) [inline documentation/2_Context_of_the_International_Data_Spaces/2_5_Meaningful_data.md:13]:
```suggestion
In any case, governance has to be put in place to make sure that a semantic standard serves the needs of the community as best as possible and will remain doing so. How these governance processes can be organized is discussed in section 4.
```
Two questions:
* "as best as possible" doesn't work. The proper adverb here would be "well". So, can one say "serves the needs of the community as _well_ as possible", or would that result in the "as well as" being mis-read as "and"? In that case, would we have an alternative idea how to phrase it?
* Do we have to write section numbers, or can we refer to other sections by ID?

File: documentation/2_Context_of_the_International_Data_Spaces/2_5_Meaningful_data.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+In any case, governance has to be put in place to make sure that a semantic standard serves the needs of the community as best as possible and will remain doing so. How these governance processes can be organized is discussed in section 4.  
+
+From a technology perspective, achieving semantic interoperability requires the use semantic technologies to create Linked Data. Semantic technologies such as OWL, RDF and SKOS allow us to enrich data with meaning by creating links to other datasets and vocabularies, enabling automatic reasoning over data through rules. The role of vocabularies and other semantic technologies in IDS is discussed in section 3.
```
Comment (clange) [inline documentation/2_Context_of_the_International_Data_Spaces/2_5_Meaningful_data.md:15]:
```suggestion
From a technology perspective, achieving semantic interoperability requires the use semantic technologies to create Linked Data. Semantic technologies such as RDF, SHACL, OWL and SKOS allow us to enrich data with meaning by creating links to other datasets and vocabularies, enabling automatic reasoning over data through rules. The role of vocabularies and other semantic technologies in IDS is discussed in section 3.
```
* added SHACL, which is also a relevant standard for semantic interoperability in the IDS.
* tried to order more logically, starting with RDF, which is the foundation of everything else.

**Review by clange — APPROVED** [review]

File: documentation/2_Context_of_the_International_Data_Spaces/2_5_Meaningful_data.md (hunk 3 above)
Comment (pieterklinker) [reply documentation/2_Context_of_the_International_Data_Spaces/2_5_Meaningful_data.md:11]: automatisation /  automation -> automisation

File: documentation/2_Context_of_the_International_Data_Spaces/2_5_Meaningful_data.md (hunk 4 above)
Comment (pieterklinker) [reply documentation/2_Context_of_the_International_Data_Spaces/2_5_Meaningful_data.md:13]: as good as possible

File: documentation/2_Context_of_the_International_Data_Spaces/2_5_Meaningful_data.md (hunk 5 above)
Comment (pieterklinker) [reply documentation/2_Context_of_the_International_Data_Spaces/2_5_Meaningful_data.md:15]: the use semantic technologies -> the use of semantic technologies

**Review by ssteinbuss — APPROVED** [review]

**Pull request merged**